src/components/post-footer/post-footer.component.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+const PostFooter = ({ voteData, showVoteResults }) => {
+  const {
+    numOfVotesForOptionA,
+    numOfVotesForOptionB,
+    totalVotes,
+    hasUserVotedForA,
+    hasUserVotedForB,
+    optionAName,
+    optionBName,
+  } = voteData;
+
+  const voteAPercent = (numOfVotesForOptionA / totalVotes) * 100 || 0;
+  const voteBPercent = (numOfVotesForOptionB / totalVotes) * 100 || 0;
+
+  /**
+   * TODO: Cleanup the way the option percentage are handled.
+   * Remove showing and hiding DOM elements and instead only mount elements to DOM when needed
```

Comment:
There is going to have to be at least one conditional on show/ hidden because of the different style if someone has %100  of the votes